src/Charcoal/Cache/Middleware/CacheMiddleware.php

```diff
@@ -131,7 +138,7 @@ public function __construct(array $data)
         $this->excludedQuery = $data['excluded_query'];
         $this->ignoredQuery  = $data['ignored_query'];
 
-        $this->skipCache = (array)$data['skip_cache'];
```

Comment:
I'm unfamiliar with this property. Why is it removed but the default value in `defaults()` preserved?

Reply:
oups